cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -427,12 +425,16 @@
               Verify that the CWT is a valid CBOR object.
             </t>
             <t>
-              If the object begins with the CWT CBOR tag, remove it
-	      and verify that one of the COSE CBOR tags follows it.
-	    </t>
-	    <t>
-	      If the object is tagged with one of the COSE CBOR tags, remove it
-	      and verify that it corresponds to the structure of the following COSE object.
+              If the object begins with the CWT CBOR tag, remove it and
+              continue verification.
```

Comment:
I´m not against mandating COSE tag when CWT tag is used, it actually makes sense I was only of the impression that it was not the agreement. Lets go for it and see it someone thinks it is bad.

I was a bit unclear I meant always as in whenever the CWT tag is used it will be used on the outer object not that one would always use it. I even think the cases for the CWT tag is few.